docs/thirdparty-rest-v1.0-OpenApi.yaml

```diff
@@ -0,0 +1,846 @@
+swagger: "2.0"
+info:
+  version: "1.1"
+  title: Open API for FSP Interoperability (FSPIOP)
+  description: Based on API Definition.docx updated on 2020-04-07 Version 1.1. Note - The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP header.
+  license:
+    name: Open API for FSP Interoperability (FSPIOP)
+basePath: /fsp
+schemes:
+- http
+- https
+
+paths:
+  /consent:
```

Comment:
since every other resource we have uses a plural, can this one be as well? or does it not make sense here?